lib/rubocop/cop/rspec/mixin/location.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
```

Comment:
What if we put this under `lib/rubocop/rspec` next to `Language` etc?
Then, tricks for RSpec/FilePath and `:cop` metadata inclusion won't be necessary.

Reply:
We do not want to use include modules, so we would like to keep mixin directory as much as possible. We want to keep it as a mixin as much as possible, because we want to call it `LocationHelp.some_method`.

If we move it to `lib/rubocop/rspec` We will need to call it `RuboCop::RSpec::LocationHelp.some_method`.
If I have missed something, please let me know.